spec/spec_helper_acceptance_local.rb

```diff
@@ -10,5 +10,25 @@
   c.before :suite do
     # Download the plugins to ensure up-to-date facts
     PuppetLitmus::PuppetHelpers.run_shell('/opt/puppetlabs/bin/puppet plugin download')
+    pp = <<-PUPPETCODE
+    file { '/etc/puppetlabs/license.key':
```

Comment:
This is a public repo. I am not sure it is a good idea to have this license public. Can we pull it in from a GitHub secret?

Reply:
will remove and add in as a secret when its needed